SPECS-EXTENDED/mock/mock.spec

```diff
@@ -0,0 +1,661 @@
+%bcond_with lint
+%bcond_without tests
+
+# mock group id allocate (Must not overlap with any other gid in Mariner)
+%global mockgid 135
+
+%global __python %{__python3}
+%global python_sitelib %{python3_sitelib}
```

Comment:
Can we replace the unversioned python macros in this spec with the python3 versions, since we're moving towards Python2 removal? Having this redefinition is a bit confusing here.